zerver/tests/test_queue_worker.py

```diff
@@ -214,32 +217,82 @@ def cancel(self) -> None:
 
     def test_mirror_worker(self) -> None:
         fake_client = self.FakeClient()
+        stream = get_stream('Denmark', get_realm('zulip'))
+        stream_to_address = encode_email_address(stream)
         data = [
             dict(
                 message=u'\xf3test',
                 time=time.time(),
-                rcpt_to=self.example_email('hamlet'),
-            ),
-            dict(
-                message='\xf3test',
-                time=time.time(),
-                rcpt_to=self.example_email('hamlet'),
-            ),
-            dict(
-                message='test',
-                time=time.time(),
-                rcpt_to=self.example_email('hamlet'),
-            ),
-        ]
+                rcpt_to=stream_to_address
+            )
+        ] * 3
```

Comment:
I probably would have put this cleanup in a prep commit before the others, just for readability of the commit series.